packages/client/src/components/QuestionResult/QuestionResult.styles.ts

```diff
@@ -0,0 +1,50 @@
+import styled, { keyframes } from 'styled-components';
+
+import { Icon } from '@codement/ui/components/Icon/Icon';
+import { theme as t } from '@codement/ui/css/theme';
+import { QuestionResultProps } from './QuestionResult';
+
+const slideUp = keyframes`
+    from {
+        opacity: 0;
+        transform: translate3d(0, 75%, 0);
+    }
+
+    to {
+        opacity: 1;
+        transform: none;
+    }
+`;
+
+export const QuestionResultDiv = styled.div`
+    animation: ${slideUp} 0.9s linear;
```

Comment:
In animations, we'd almost never use `linear` as it doesn't look the greatest. 
Let's bring the speed down too!
Perhaps we use `0.3s ease-out`?

Reply:
Got it! Will make the necessary changes to the animation.